lib/hammer_cli/output/fields.rb

```diff
@@ -19,6 +19,7 @@ def hide_blank?
     end
 
     def display?(value)
+      return false if HammerCLI.context[:verbosity].to_i < self.parameters[:verbosity].to_i
```

Comment:
This chunk of code is repeated at multiple places. Could you please extract it into a method and move it to the parent class?